test/exe/require_glibc.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	version "github.com/hashicorp/go-version"
+)
+
+func main() {
+	required, _ := version.NewVersion("2.27")
+	args := os.Args[1:]
+	cmd := exec.Command("objdump", "-T", args[0])
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		log.Fatal("failed to check GLIBC version:", err)
```

Comment:
Log line usually starts with capital letter. (line 31 as well)

Reply:
I don't know about this "usual" thing. I'll update it.